tests/unit/libexpr/nix_api_value.cc

```diff
@@ -26,11 +31,16 @@ TEST_F(nix_api_expr_test, nix_value_set_get_int)
     ASSERT_EQ(NIX_TYPE_INT, nix_get_type(ctx, value));
 }
 
-TEST_F(nix_api_expr_test, nix_value_set_get_float)
+TEST_F(nix_api_expr_test, nix_value_set_get_float_invalid)
 {
     ASSERT_FLOAT_EQ(0.0, nix_get_float(ctx, nullptr));
-    ASSERT_DEATH(nix_get_float(ctx, value), "");
+    assert_ctx_err();
+    ASSERT_FLOAT_EQ(0.0, nix_get_float(ctx, value));
+    assert_ctx_err();
+}
 
+TEST_F(nix_api_expr_test, nix_value_set_get_float)
+{
     float myDouble = 1.0;
```

Comment:
Unrelated, but this freaked me out for a sec. It doesn't seem to serve a purpose.

```suggestion
    double myDouble = 1.0;
```

Reply:
oops, good catch, addressed it here: f7da54465eaba0081642c9f8892ef81a548d8f14